src/loaders/OmeZarrLoader.ts

```diff
@@ -519,7 +554,7 @@ class OMEZarrLoader implements IVolumeLoader {
         // For nearby timesteps: duplicate only the load requests made on the current timestep on this one too
         for (const coord of chunkCoords) {
           const [c, z, y, x] = coord.slice(1);
-          this.prefetchChunk(scaleLevel.path, [t, c, z, y, x], subscriber);
+          this.prefetchChunk(scaleLevel.path, [t, c, z, y, x], subscriber, Math.abs(t - activeTimestep) * 2000);
```

Comment:
just for readably: could define this as `const delayMs = Math.abs(t - activeTimestep) * 2000`. If I was looking at this in VScode I would be able to see the the inputs to `prefetchChunk`, but here it's hard to tell what this argument is

Reply:
agree.. probably will make the 2000 a named constant too.  For now this is just being tweaked to see what delay values work well for what kinds of data sizes.